### DISCUSSION TITLE
"Sending of the message failed" with incomplete email address

### DISCUSSION THREAD

**Discussion by mcumminsspusa** [Extended-Thunder/send-later#537] [Q&A] [answered]
I am using Send-later Version 9.2.9 and Thunderbird Version 102.10.0.  

I composed an email and entered a partial email address in the To: field (xxxxx@gmail. ),  I forgot to complete the email address after composing my message and clicked Send Later.  As expected, I received the following  error message : 

"Sending of the message failed.
An error occurred while sending mail.  The amil server responded:
Requested action not taken: mailbox unavailable
invalid DNS MX or A/AAAA resource record.
Please check the message recipient "xxxxx@gmail." and try again."

The problem is that I continue to receive this error message each time I send an email using Send Later.  The email with the incomplete address has been deleted from the Drafts folder.  How do I clean up the system to prevent receiving this error message?

**Comment by Franvgls** [comment]
Hello, 
Have more or less the same issue, in my case I added an address that had wrong in the to field and I keep on receiving the same message mentioned above, apart from that thunderbird works fine, but cant get ride fo the sending error. 
Also using the 102.11.2 64 bits version, but the error comes from a couple of updates at least.
Any idea on how to stop receiving the message?
Best, 
Fran

**Comment by jikamens** [comment, marked as answer]
When Send Later sends a scheduled message, it puts the message in your Outbox folder (underneath Local Folders) and then tells Thunderbird to send unsent messages in Outbox. If there's something wrong with the message then it'll get stuck in your Outbox and Thunderbird will try to send it each time Send Later delivers another scheduled message. To fix, this open the Outbox folder under Local Folders and delete the bad message(s).
